tools/api-markdown-documenter/src/api-item-transforms/helpers/TableHelpers.ts

```diff
@@ -734,9 +734,18 @@ function createTypeExcerptCell(
  */
 function getTableHeadingTitleForApiKind(itemKind: ApiItemKind): string {
 	switch (itemKind) {
+		case ApiItemKind.CallSignature: {
+			return "Call Signature";
+		}
+		case ApiItemKind.ConstructSignature: {
```

Comment:
Seems like we don't have any test collateral for Constructors? Would be nice to add at some point.

Reply:
For constructor signatures specifically, yeah. I'll add it to my list. Hoping to do a PR later today that adds a bit more coverage to our test suites. Trying to find a good way to do that without bloating it even further is the tricky part :/